tests/functional/ScaffolderTest.php

```
@@ -474,6 +491,117 @@ public function testLogsSkipMessageWhenRootPackageHasNoScaffoldExtra(): void
+                        'app.php' => [
+                            'source' => 'stubs/app.php',
+                            'mode' => 'replace',
+                        ],
+                    ],
+                ],
+            ],
+        );
+        // 'bad' is listed first so its manifest-load failure triggers the continue-branch before 'good' is processed.
+        $root = $this->makeRootPackage(['yii2-extensions/bad', 'yii2-extensions/good']);
+
+        $io = new BufferIO();
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $io. Configured minimum length is 3. (undefined)

(ShortVariable)